ui/Screen/Onboarding/Success.svelte

```diff
@@ -45,8 +45,8 @@
     <p
       style="text-align: center; width: 23.13rem; margin: 1.75rem 0 1.75rem 0;
       color: var(--color-foreground-level-6);">
-      This is your handle — it's unique to you!<br /> Click to copy it and share
-      with others so that they can find you.
+      This is your peerID — it's unique to your machine!<br /> Click to copy it and
+      share with others so that they can add you as a remote of a project.
```

Comment:
Now the copy is talking about `PeerId` but the actual thing shown in the UI is a `URN`.

Reply:
<img width="627" alt="Screenshot 2020-10-30 at 15 47 25" src="https://user-images.githubusercontent.com/158411/97719321-3da57600-1ac7-11eb-92c3-a123900fdbbe.png">

Reply:
Didn't we at one time agree that we were going to refer to this as a handle? Did we decide to go back to peerID? @cloudhead @brandonhaslegs 

If so, lmk bc I have to update the glossary

Reply:
ok undid this. But this is basically pointless right now. Cause users can't do anything with their urn... So copy is correct for the useless thing.